back/app/controllers/web_api/v1/phases_controller.rb

```diff
@@ -10,9 +10,9 @@ def index
       .where(project_id: params[:project_id])
       .order(:start_at)
     @phases = paginate @phases
-    @phases = @phases.includes(:permissions, :report, :custom_form)
+    @phases = @phases.includes(:permissions, :report, :custom_form, :manual_voters_last_updated_by)
 
-    render json: linked_json(@phases, WebApi::V1::PhaseSerializer, params: jsonapi_serializer_params, include: %i[permissions])
+    render json: linked_json(@phases, WebApi::V1::PhaseSerializer, params: jsonapi_serializer_params, include: %i[permissions manual_voters_last_updated_by])
```

Comment:
I wonder whether we need to include it. I understand that if we don't we risk an N+1, but the way this is displayed today in the front-end, I think we never actually show how who updated last in a listing view.

If we don't include it, we also don't need to include it (right?) in the database query above and can avoid the extra query.

Acceptable the way it is now, not blocking.